pages/dkp/kommander/1.4/tutorials/login-cluster/index.md

```diff
@@ -22,34 +22,34 @@ If your group can access a cluster managed by Kommander, you can connect to that
 
 In this example, the Kommander cluster has two managed clusters on Azure but, as a user, you can only access the management cluster.
 
-1. Go to the landing page, provided by an administrator, and select **Generate kubectl Token**.
+1. Go to the landing page, provided by an administrator, and select **Generate Kubectl Token**.
 
-![Konvoy Landing Page](/dkp/kommander/1.4/img/konvoy-landing-page.png)
+   ![Konvoy Landing Page](/dkp/kommander/1.4/img/konvoy-landing-page.png)
 
-2. Select the cluster you want to log into.
+1. Select the cluster you want to log into.
 
-![Generate Kubernetes Token](/dkp/kommander/1.4/img/generate-kubernetes-token.png)
+   ![Generate Kubernetes Token](/dkp/kommander/1.4/img/generate-kubernetes-token.png)
 
-3. As a user having access to the management cluster, select the `Main` cluster and log in using an identity provider.
+1. As a user having access to the management cluster, and wanting to generate a kubeconfig for this cluster, select the `Main` cluster and log in using an identity provider.
```

Comment:
I am having troubles understanding what this sentence means.  Isn't the first part of this sentence redundant with the task description at the top of this procedure?   

Should we instead say:

```suggestion
1. Log in using an identity provider.
```
???

I also don't understand what `Main` cluster means here - is this different than the cluster I just selected in the previous step?

Reply:
Yeah, it's written a little clunky, that's one reason why I wanted to re-work this page... Let me re-work a bit more.

Reply:
Resolving this - just made a push - can you take a look again @cneth?